app/controllers/applications_controller.rb

```diff
@@ -1,14 +1,39 @@
 class ApplicationsController < ApplicationController
-  before_action :require_admin, only: [:admin_destroy]
   before_action :get_event
   before_action :get_application, except: [:new, :create, :continue_as_guest]
   before_action :ensure_correct_user, only: [:show, :edit]
-  before_action :skip_validation, only: [:save_draft, :approve, :reject, :revert]
   skip_before_action :require_login, only: [:new, :create, :continue_as_guest]
 
+  def new
+   if @event.application_process == 'application_by_organizer'
```

Comment:
Would it be possible to only link to this controller action, only if you're allowed to create a new application? Like not showing the link in that case? You could then delete this conditional.